src/Screens/HomeScreen/HomeScreen.tsx

```diff
@@ -8,26 +8,42 @@ import { useTranslation } from "react-i18next";
 import { fetchTodos } from "store/todo/todoActions";
 import { useDispatch } from "react-redux";
 import Form from "components/Form/Form";
+import { StyleSheet } from "aphrodite";
+import CustomButton from "components/CustomButton/CustomButton";
 
 const style = bemCssModule(HomeScreenStyles);
 
 const HomeScreen = () => {
   const { t } = useTranslation();
   const dispatch = useDispatch();
-  const btnAddTask = <button>Add task</button>;
+  const [showForm, setShowForm] = useState(false);
 
   useEffect(() => {
     dispatch(fetchTodos());
   }, [dispatch]);
 
   return (
     <LayoutWrapper>
-      <HeaderPanel actionButton={btnAddTask} />
+      <HeaderPanel />
+      <CustomButton
+        label="Add task"
+        onClick={() => setShowForm(!showForm)}
+        filled={false}
```

Comment:
Filled should be false by default in your CustomButton. If you pass filled props this should change your button styles into filled (for example with your primary color)  or if not into outlined styles (e.g with only borderColor and primary font color)